models/issue.go

```diff
@@ -1928,3 +1928,16 @@ func (issue *Issue) updateClosedNum(e Engine) (err error) {
 	}
 	return
 }
+
+// UpdateIssuesMigrations updates issues' migrations information
+func UpdateIssuesMigrations(repoID, originalAuthorID, posterID int64) error {
+	_, err := x.Table("issue").
+		Where("repo_id = ?", repoID).
+		And("original_author_id = ?", originalAuthorID).
+		Update(map[string]interface{}{
+			"poster_id":          posterID,
+			"original_author":    "",
+			"original_author_id": 0,
```

Comment:
This functions seems to be about clearing up the original author information, rather than update it. Which is it?

Reply:
`poster_id` stores local user id and `original_author_id` stores github's user id.
This function will convert an issue's poster from an original author to a local user if we have already known his local user is related a github account. All related information are stored on `external_login_user`. So we could know which local user id is point to a github user id.

Reply:
Of course, I missed the relevant line.